reference/shipping_provider.yml

```
@@ -428,16 +402,14 @@ components:
       description: Payload sent off to a Shipping Provider in order to get quotes.
       properties:
         base_options:
-          $ref: "#/components/schemas/BaseOptions"
+          $ref: '#/components/schemas/BaseOptions'
         zone_options:
           type: object
-          description: Optional, any zone specific request options declared by the carrier
-            and configued by the merchant to retrieve rates
+          description: 'Optional, any zone specific request options declared by the carrier and configured by the merchant to retrieve rates.'
```

```suggestion
          description: Any zone-specific request options declared by the carrier and configured by the merchant to retrieve rates. Optional.
```
remove comma splice, add hyphen.